lib/oppen/pretty_printer.rb

```diff
@@ -10,6 +10,44 @@ module Oppen
   # PrettyPrinter class responsible for scanning tokens and passing
   # them to the printer.
   class PrettyPrinter
+    # Ring buffer left indexe.
```

Comment:
`indexe` is not an english word

Reply:
Hah. I am famous for this. I will install [typos](https://github.com/crate-ci/typos/) in the CI.